frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/store.ts

```diff
@@ -0,0 +1,69 @@
+export enum Providers {
+  AWS = 'AWS S3',
+  AZURE = 'Azure Blob',
+  S3 = 'S3 Compatible',
+  PVC = 'PVC',
+}
+
+export const testName = 'test-bucket';
+
+export enum StoreType {
+  BackingStore = 'backingstore',
+  NamespaceStore = 'namespacestore',
+}
```

Comment:
Move all enums together

Reply:
if you mean to move the storetype enum from store.ts to bc.ts does not make sense to have something related to BackingStore and NamespaceStore to be in bucket class module or vice versa